source/common/json/config_schemas.cc

```diff
@@ -161,6 +161,15 @@ const std::string Json::Schema::LISTENER_SCHEMA(R"EOF(
         "required": ["cert_chain_file", "private_key_file"],
         "additionalProperties": false
       },
+      "listener_filters" : {
```

Comment:
Please do not support this feature in v1.

Reply:
Removed.